mods/bucket/locale/bucket.es.tr

```diff
@@ -0,0 +1,5 @@
+# textdomain: bucket
+Empty Bucket=Balde vacío
```

Comment:
I think `Cubeta` sounds better.
```suggestion
Empty Bucket=Cubeta vacía
```

Reply:
Are you from Latin America?
In Spain the term 'cubeta' is a small bucket. The correct word would be 'cubo'. 'Balde' in Spain is a old-fashioned word.

Reply:
In my country, `Cubeta` is the correct word.
`Cubo` sounds like other thing.

BTW: I have never heard about `Balde`.
This can be solved, as it can be written in 3 ways or even more ways.